src/lib.rs

```diff
@@ -309,17 +357,18 @@ macro_rules! ct_constant_copy_gen {
             for i in 0..x_len {
                 let y_temp = y[i].clone();
                 let x_temp = x[i].clone();
-                x[i] = $copy_symbol(flag,y_temp,x_temp); 
+                x[i] = <$code as ConstantTime>::ct_select(flag,y_temp,x_temp);
```

Comment:
Your comment https://github.com/valarauca/consistenttime/issues/5#issuecomment-272042450 indicates that maybe this line should be reverted to `$copy_symbol`, which entails adding `copy_symbol` back.